docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -133,6 +133,8 @@ Property                                              Description
                                                       ``BIGNUMERIC`` and ``TIMESTAMP`` types are unsupported.        ``false``
 ``bigquery.views-cache-ttl``                          Duration for which the materialization of a view will be       ``15m``
                                                       cached and reused. Set to ``0ms`` to disable the cache.
+``bigquery.dataset-cache-ttl``                        Duration for which the list of datasets from BigQuery          ``1m``
+                                                      are cached and reused. Set to ``0ms`` to disable the cache.
```

Comment:
What config name should be used when we introduce cache for tables? bigquery.table-cache-ttl?

Reply:
Agreed. We should just use `metadata-cache-ttl` maybe?

Reply:
+1, will change to `metadata-cache-ttl`